### ISSUE TITLE
Settings?

### ISSUE THREAD

**Issue by sheibeck** [Draggable/formeo#13]
Just curious if the Settings button is currently not ready, or if it's an issue. When I click it, it doesn't do anything. I'm assuming it should (or will) open up some form wide settings that you can specify.

And, on this note, wondering if the settings will be something extendable so adding custom settings to a particular form would be doable.

**Comment by kevinchappell** [comment]
@sheibeck Yes on both counts. The stage where fields reside will be hidden when form settings panel is open. Custom settings will be defined using the same format as custom fields.

One thing to note is that given the extensibility of this part of the builder, developers should have a whitelist of acceptable settings to check against submitted data before its saved.

**Comment by sheibeck** [comment]
Awesome! Looking forward to it.
